src/persist-client/src/read.rs

```diff
@@ -1009,11 +1095,11 @@ mod tests {
 
     #[test]
     fn client_exchange_data() {
-        // The whole point of ReaderEnrichedHollowBatch is that it can be exchanged between
+        // The whole point of LeasedBatch is that it can be exchanged between
```

Comment:
isn't this SerdeLeasedBatch now?

Reply:
Was in a `WIP` wrapper so didn't actually run; fixed